src/test/kotlin/no/nav/syfo/veileder/api/VeiledereApiTest.kt

```diff
@@ -28,11 +34,25 @@ import org.junit.jupiter.api.Test
 class VeiledereApiTest {
     private val basePath = "/syfoveileder/api/v3/veiledere"
 
-    private lateinit var externalMockEnvironment: ExternalMockEnvironment
+    private val externalMockEnvironment: ExternalMockEnvironment = ExternalMockEnvironment()
+
+    val azureAdClient = AzureAdClient(
+        azureAppClientId = externalMockEnvironment.environment.azureAppClientId,
+        azureAppClientSecret = externalMockEnvironment.environment.azureAppClientSecret,
+        azureOpenidConfigTokenEndpoint = externalMockEnvironment.environment.azureOpenidConfigTokenEndpoint,
+        graphApiUrl = externalMockEnvironment.environment.graphapiUrl,
+        cache = externalMockEnvironment.valkeyCache,
+        httpClient = externalMockEnvironment.mockHttpClient,
+    )
+    val graphApiClient = GraphApiClient(
+        azureAdClient = azureAdClient,
+        baseUrl = externalMockEnvironment.environment.graphapiUrl,
+        cache = externalMockEnvironment.valkeyCache,
+        httpClient = externalMockEnvironment.mockHttpClient,
+    )
```

Comment:
Alternativt kunne vel denne vært 
```kotlin
val graphApiClient = mockk<GraphApiClient>
```
men vet ikke om du prøvde det? Må du lage originalen for å kunne bruke funksjonene i companion-objectet?